src/test/java/uk/gov/hmcts/darts/courthouse/service/CourthouseServiceImplTest.java

```diff
@@ -0,0 +1,136 @@
+package uk.gov.hmcts.darts.courthouse.service;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.ArgumentCaptor;
+import org.mockito.Captor;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.hmcts.darts.common.entity.Courthouse;
+import uk.gov.hmcts.darts.courthouse.CourthouseRepository;
+import uk.gov.hmcts.darts.courthouse.mapper.CourthouseToCourthouseEntityMapper;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyInt;
+
+@ExtendWith(MockitoExtension.class)
+class CourthouseServiceImplTest {
+
+    public static final String TEST_COURTHOUSE_NAME = "Test courthouse";
+    public static final short CODE = 123;
+    public static final int COURTHOUSE_ID = 11;
+
+    @InjectMocks
+    CourthouseServiceImpl courthouseService;
+
+    @Mock
+    CourthouseRepository repository;
+
+    @Mock
+    CourthouseToCourthouseEntityMapper mapper;
+
+    @Captor
+    ArgumentCaptor<Integer> captorInteger;
+
+
+    @BeforeEach
+    void setUp() {
```

Comment:
No need for this

Reply:
Why not ? we need to clean the database before each test to make sure they don't interfere.

Reply:
I thought the same @HMCTSDeveloper - it's empty!

Reply:
@HMCTSDeveloper  If you want it to clean the database, then you could use a deleteAll command on the courthouse respository to do it.

e.g.
```java
    @BeforeEach
    void beforeEach() {
        repository.deleteAll();
    }
```